### PR TITLE
fix(command-dev): pass functions server uri to traffic mesh

### PR DESCRIPTION
**Pull request by erezrokah**
Adds support for functions redirects when using `--trafficMesh`.

Pending a new release of https://github.com/netlify/traffic-mesh-agent

### CODE REVIEW

**Comment by calavera** [comment]
I tagged a new version that should include this new flag: https://github.com/netlify/traffic-mesh-agent/releases/tag/v0.19.0

**Review by calavera — APPROVED** [review]

**Review by mraerino — APPROVED** [review]

**Review by calavera — CHANGES_REQUESTED** [review]
I'm blocking this because the new version of the agent is broken. I'm looking into it.

File: src/utils/traffic-mesh.js [hunk 1]
```diff
@@ -33,10 +33,14 @@ const installTrafficMesh = async ({ log }) => {
   })
 }
 
-const startForwardProxy = async ({ port, frameworkPort, projectDir, log, debug }) => {
+const startForwardProxy = async ({ port, frameworkPort, functionsPort, projectDir, log, debug }) => {
   await installTrafficMesh({ log })
   const args = ['start', '--port', port, '--forward-proxy', `http://localhost:${frameworkPort}`, '--watch', projectDir]
```
Comment (calavera) [inline src/utils/traffic-mesh.js:38]: The command has changed, it should be `start local` when 0.19.1 is public(currently building):
```suggestion
  const args = ['start', 'local', '--port', port, '--forward-proxy', `http://localhost:${frameworkPort}`, '--watch', projectDir]
```

Comment (calavera) [reply src/utils/traffic-mesh.js:38]: https://github.com/netlify/traffic-mesh-agent/releases/tag/v0.19.1

Comment (erezrokah) [reply src/utils/traffic-mesh.js]: Redirecting to functions is still broken. Looks like it's capturing the redirect property, but redirecting to the `--forward-proxy` port:

```
{"timestamp":"Sep 06 16:53:53.197","level":"DEBUG","spans":[{"bundle_id":"","host":"\"localhost:8888\"","path":"\"/.netlify/functions/index\"","name":"find remap"},{"path":"\"/.netlify/functions/index\"","req":"Request { method: \"GET\", scheme: \"\", host: \"localhost:8888\", path: \"/.netlify/functions/index\", raw_query: \"\", fragment: None, headers: {\"host\": \"***\", \"accept\": \"***\", \"user-agent\": \"***\"}, cookies: {} }","name":"find_lambda_proxy_headers"}],"span":{"path":"\"/.netlify/functions/index\"","req":"Request { method: \"GET\", scheme: \"\", host: \"localhost:8888\", path: \"/.netlify/functions/index\", raw_query: \"\", fragment: None, headers: {\"host\": \"***\", \"accept\": \"***\", \"user-agent\": \"***\"}, cookies: {} }","name":"find_lambda_proxy_headers"},"target":"traffic_mesh","fields":{"message":"looking for a remap or a lambda proxy call"}}
{"timestamp":"Sep 06 16:53:53.197","level":"DEBUG","target":"traffic_mesh::proxy","fields":{"message":"proxying request","uri":"http://localhost:3999/.netlify%2Ffunctions%2Findex"}}
```

**Comment by erezrokah** [comment]
Regardless I released this in https://github.com/netlify/cli/releases/tag/v2.61.1

**Pull request merged**